platform/linux-generic/odp_schedule_basic.c

```diff
@@ -1938,13 +2039,72 @@ static odp_schedule_group_t schedule_group_create(const char *name,
 
 			grp_update_mask(i, mask);
 			group = (odp_schedule_group_t)i;
-			spread_thrs_inc(group, thr_tbl, count);
+			spread_thrs_inc(group, thr_tbl, num_thr);
 			sched->sched_grp[i].allocated = 1;
+			sched->num_grps++;
 			break;
 		}
 	}
 
+	return group;
+}
+
+static odp_schedule_group_t schedule_group_create(const char *name,
+						  const odp_thrmask_t *mask)
+{
+	odp_schedule_group_t group;
+	int count;
+
+	count = odp_thrmask_count(mask);
+	if (count < 0) {
+		_ODP_ERR("Bad thread count\n");
+		return ODP_SCHED_GROUP_INVALID;
+	}
+
+	int thr_tbl[ODP_THREAD_COUNT_MAX];
+
+	if (count && threads_from_mask(thr_tbl, count, mask))
+		return ODP_SCHED_GROUP_INVALID;
+
+	odp_ticketlock_lock(&sched->grp_lock);
+	group = allocate_group(name, mask, thr_tbl, count);
+	odp_ticketlock_unlock(&sched->grp_lock);
+
+	return group;
+}
+
+static odp_schedule_group_t schedule_group_create_2(const char *name,
+						    const odp_thrmask_t *mask,
+						    const odp_schedule_group_param_t *param)
+{
+	odp_schedule_group_t group;
+	int count;
+
+	if (param->prio.min < sched->config_if.min_prio ||
```

Comment:
According to API `param->prio.min` should be ignored if `param->prio.num` is zero.